acceptance/status_server_test.go

```diff
@@ -80,6 +80,7 @@ func checkNode(t *testing.T, client *http.Client, node *localcluster.Container,
 // TestStatusServer starts up an N node cluster and tests the status server on
 // each node.
 func TestStatusServer(t *testing.T) {
+	t.Skipf("Test is flaky - will fix - Bram.")
```

Comment:
Adding `TODO(bram)` to the start of the comment make it easier to grep the codebase for TODOs.